### PR TITLE
Toby ylim

### PR DESCRIPTION
**Pull request by tdhock**
this branch has a small fix for using ylim in one of the #161 examples

http://www.cookbook-r.com/Graphs/Bar_and_line_graphs_%28ggplot2%29/line-point-ylim.png

### CODE REVIEW

**Comment by tdhock** [comment]
@mkcor @chriddyp @cpsievert I updated the test table and saw no regressions

http://ropensci.github.io/plotly-test-table/tables/618456d6e5d71de59481c5b1681db793a77f26c3/index.html

I added this new test which now passes (error on master)

http://ropensci.github.io/plotly-test-table/tables/618456d6e5d71de59481c5b1681db793a77f26c3/ylim-one-trace.html

please code review

File: R/ggplotly.R [hunk 1]
```diff
@@ -305,12 +305,28 @@ gg2list <- function(p){
...
+    ax.list$titlefont <- theme2font(title.text)
+    ax.list$type <- if(misc$is.continuous[[xy]]){
+      "linear"
+    }else if(misc$is.discrete[[xy]]){
+      "category"
+    }else if(misc$is.date[[xy]] || misc$is.datetime[[xy]]){
+      "date"
+    }else{
+      stop("unrecognized data type for ", xy, " axis")
+    }
```
Comment (chriddyp) [inline R/ggplotly.R]: :lipstick: - spaces around `if`: 
image

File: R/ggplotly.R [hunk 2]
```diff
@@ -305,12 +305,28 @@ gg2list <- function(p){
...
+    ## (trace.order.list is only used for type=category).
+    title.text <- e(s("axis.title.%s"))
+    ax.list$titlefont <- theme2font(title.text)
+    ax.list$type <- if(misc$is.continuous[[xy]]){
+      "linear"
+    }else if(misc$is.discrete[[xy]]){
+      "category"
+    }else if(misc$is.date[[xy]] || misc$is.datetime[[xy]]){
+      "date"
+    }else{
```
Comment (chriddyp) [inline R/ggplotly.R]: I'm assuming `ggplot2` doesn't have a `log` type? (that it is just `continuous`)

**Comment by chriddyp** [comment]
cool! besides that super minor style issue, all good from me. bump version and :+1:

**Pull request merged**